templates/flower/flower-sidecar.yaml

```diff
@@ -0,0 +1,110 @@
+#################################
+## Flower UI sidecar ConfigMap ##
```

Comment:
```suggestion
## Flower UI auth sidecar ConfigMap ##
```

And filename to `flower-auth-sidecar-configmap.yaml`.

Reply:
file renamed as per suggestion.